splash/browser_tab.py

```diff
@@ -1095,6 +1121,16 @@ def log(self, message, min_level=None):
         log.msg(message, system='render')
 
 
+class ElementsStorage(QObject):
+    def __init__(self, parent):
+        self.name = str(uuid.uuid1())
+        super(ElementsStorage, self).__init__(parent)
+
+    @pyqtSlot(name="getId", result=str)
+    def get_id(self):
+        return str(uuid.uuid1())
```

Comment:
`get_id` is used in JS for retrieving a unique ID for the element. However, it's a good idea to call the same function in the constructor.